src/Controller/CategoriesController.php

```diff
@@ -51,12 +52,31 @@ public static function create(ContainerInterface $container) {
    *   Json Array with uuid's.
    */
   public function list(string $type) {
+    $user = $this->currentUser();
+    $pattern = 'virtual_y';
+    // Get list of vy roles for current user.
+    $y_roles = array_filter($user->getRoles(), function ($role) use ($pattern) {
+      return strpos($role, $pattern) !== FALSE;
+    });
+
     $query = $this->database->select('node__field_gc_video_category', 'n');
+    $query->leftJoin('node_field_data', 'nd', 'n.entity_id = nd.nid');
```

Comment:
![image](https://user-images.githubusercontent.com/3023950/96572437-7f6b3b00-12d5-11eb-868a-b2299f9d4c98.png)
We can do this small fix.

Reply:
Fixed